daily_coding_problem_16_20.py

```diff
@@ -50,7 +50,7 @@ def get_last(self, num):
 
 
 def coding_problem_17(path_str):
-    r"""
```

Comment:
using r"""raw strings""" is necessary for me, otherwise doctest barfs

Reply:
Oh, yep, my bad. I forgot there are double quotes used in docstring already.